client/src/components/FlightAccordion.js

```diff
@@ -56,15 +56,20 @@ const FlightAccordion = ({ quote, cities }) => {
 	);
 	const departingCarrierLogo = findDepartingCarrier.LogoUrl;
 	const departingCarrierName = findDepartingCarrier.Name;
-	const findReturningCarrier = () => {
-		if (quote.InboundLeg) {
-			return carriers.find(
-				(carrier) => carrier.CarrierId === quote.InboundLeg.CarrierId
-			);
-		}
-	};
-	const returningCarrierLogo = findReturningCarrier.LogoUrl;
-	const returningCarrierName = findReturningCarrier.Name;
+
+	//Commented code out cause I wasn't sure what their future use will be
+
+	// const returningCarrierLogo = findReturningCarrier.LogoUrl;
+	// const returningCarrierName = findReturningCarrier.Name;
+	
+	// const findReturningCarrier = () => {
+	// 	if (quote.InboundLeg) {
+	// 		return carriers.find(
+	// 			(carrier) => carrier.CarrierId === quote.InboundLeg.CarrierId
+	// 		);
+	// 	}
+	// };
+
```

Comment:
Delete without fear, if it was present in a commit, you can easily get it back with `git` 🙂.
You can do: `git checkout <commit-where-code-was>` open the file with the section of code you had deleted and copy the code. Then go back to where the branch you were working on by doing `git switch -`, and pasting the code you had copied where you want it.